frontends/web/src/api/account.ts

```diff
@@ -184,6 +185,9 @@ export const getTransactionList = (code: AccountCode): Promise<ITransaction[]> =
   return apiGet(`account/${code}/transactions`);
 };
 
+export const getTransaction = (code: AccountCode, id: ITransaction['txID']): Promise<ITransaction> => {
```

Comment:
should be `internalID` not `txID`?

interesting trick, didn't know could could reference a type in an interface like that.

Reply:
You're right, I'll change it! Thx